lib/webpacker/configuration.rb

```diff
@@ -36,6 +36,10 @@ def source
       fetch(:source_path)
     end
 
+    def compile?
+      fetch(:compile).nil? ? Webpacker.env.test? : fetch(:compile)
```

Comment:
One concern is for folks upgrading for whom the `compile_missing_packs` entry would be missing... in this case, they would have to add in the values to get the expected behavior back in `test`. Should we add a warning or upgrade note in README?